<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLineEdit" name="name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="status">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>В зернах</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="url"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lable_2">
      <property name="text">
       <string>Описание url</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="deef"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lable_s">
      <property name="text">
       <string>Обжарка</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>true</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>false</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lflf">
      <property name="text">
       <string>Название</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lable">
      <property name="text">
       <string>Цена</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="price"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Объем</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
